src/components/StudioTopbar/StudioTopbar.style.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Link } from 'react-router-dom'
+import { breakpoints, colors, sizes, transitions, typography } from '@/shared/theme'
+import { TopbarBase, Avatar, Text, Button, TOP_NAVBAR_HEIGHT, TopbarBaseProps } from '@/shared/components'
+
+type ChannelInfoProps = {
+  active?: boolean
+}
+
+type NavDrawerProps = {
+  active?: boolean
+}
+
+type DrawerButtonProps = {
+  isActive?: boolean
+}
```

Comment:
`active`, `active` and then `isActive` 😅  To avoid repetition, maybe you could replace these with something like `CommonTopbarProps`